lib/Ed25519KeyPair.js

```diff
@@ -78,6 +78,62 @@ class Ed25519KeyPair extends LDKeyPair {
    * @returns {Promise<Ed25519KeyPair>} Generates a key pair.
    */
   static async generate(options = {}) {
+    // FIXME: determine exact release that included ed25519 key generation
+    if(env.nodejs && require('semver').gte(process.version, '12.0.0')) {
```

Comment:
I think this style will try and pull in `semver`, a `process` polyfill, and the `./*Node12.js` files when bundling via webpack.  Need to check what it's doing and maybe can just add package.json browser fields to disable non-browser things.

Reply:
@davidlehn I just had to make a couple mods to fix karma tests.  If karma works does that mean this concern is addressed or is further testing required?